crates/ibc/src/core/context.rs

```diff
@@ -559,6 +571,105 @@ mod val_exec_ctx {
         fn log_message(&mut self, message: String);
     }
 
+    fn chan_open_init_validate<ValCtx>(
+        ctx_a: &ValCtx,
+        module_id: ModuleId,
+        msg: MsgChannelOpenInit,
+    ) -> Result<(), ContextError>
+    where
+        ValCtx: ValidationContext,
+    {
+        chan_open_init::validate(ctx_a, &msg)?;
+        let chan_id_on_a = ChannelId::new(ctx_a.channel_counter()?);
+
+        let module = ctx_a
+            .get_route(&module_id)
+            .ok_or(ChannelError::RouteNotFound)?;
+        let _ = module.on_chan_open_init_validate(
```

Comment:
can we drop `let _ = `?

Reply:
[d6fc4c9](https://github.com/cosmos/ibc-rs/pull/352/commits/d6fc4c965d2756449d2933968648f79b86be2aaf)